public/app/features/dashboard-scene/scene/NavToolbarActions.tsx

```diff
@@ -69,7 +87,7 @@ export function ToolbarActions({ dashboard }: Props) {
 
   toolbarActions.push({
     group: 'icon-actions',
-    condition: uid && !editview && !isEditingPanel,
+    condition: uid && !editview && !isEditingPanel && !isEditing,
```

Comment:
I assumed that starring dashboards should not be visible in edit mode from what I've seen in the designs. Let me know if this is alright.

Reply:
this comment is referring to a line 89 which is a part of `Switch to old dashboard page` button. Is this intended? I think you are referring to the action above, but there's a change here so I just wanted to double check 😄

Reply:
Indeed, thank you for pointing that out! I thought I was referring the mark favorites action but I actually commented on the old dashboard switch :D But yes, both were intended modifications. I just want to double check if this is the plan: to hide those actions in edit mode.